source/guides/getting-started/introduction.html.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introduction
+next_guide:
+  title: Quick Start
+  url: /guides/getting-started/quick-start.html
+---
+
+## Introduction
+
+BigTest is a collection of packages that aim to make it easy and fast to
+acceptance test single page apps (SPAs). The packages that make up
+BigTest are:
+
+- [Convergence](/docs/convergence/)
+- [Interactor](/docs/interactor/)
+- [CLI](https://github.com/bigtestjs/cli)
```

Comment:
The quick write ups below are great, but people will miss it if they bounce after the first two sentences. Let's link to docs (and guides where available) in each section instead and either make these TOC links to the sections below, or remove these links all together.